added ::marker to generated content (#22047)
mdn/content: files/en-us/learn/css/howto/generated_content/index.md

### Text content

> CSS can insert text content before or after an element. It can also be used to change the content of a marker before a {{HTMLElement('li')}} or other elements styled as a {{ cssxref("display", "list-item") }}. To specify this, make a rule and add {{ cssxref("::before") }}, {{ cssxref("::after") }}, or {{cssxref("::marker")}} to the selector. In the declaration, specify the {{ cssxref("content") }} property with the text content as its value.

Reviewer: "an" `<li>`? (fly by)

Reviewer 2: I pronounce this as "lee" 😄

Author: I need to ask someone who knows way more than me: @dipikabh?

Reviewer: I don't "know" either. Last time I had a similar argument I was wrong. So don't block on this.

> CSS can insert text content before or after an element. It can also be used to change the content of a marker before a {{HTMLElement('li')}} or other elements styled as a {{ cssxref("display", "list-item") }}. To specify this, make a rule and add {{ cssxref("::before") }}, {{ cssxref("::after") }}, or {{cssxref("::marker")}} to the selector. In the declaration, specify the {{ cssxref("content") }} property with the text content as its value.

Author: ```suggestion
CSS can insert text content before or after an element. It can also be used to change the content of a marker before a {{HTMLElement('li')}} or other element with {{ cssxref("display", "display: list-item;") }}. To specify this, make a rule and add {{ cssxref("::before") }}, {{ cssxref("::after") }}, or {{cssxref("::marker")}} to the selector. In the declaration, specify the {{ cssxref("content") }} property with the text content as its value.
```

> CSS can insert text content before or after an element. It can also be used to change the content of a marker before a {{HTMLElement('li')}} or other element with {{ cssxref("display", "display: list-item;") }}. To specify this, make a rule and add {{ cssxref("::before") }}, {{ cssxref("::after") }}, or {{cssxref("::marker")}} to the selector. In the declaration, specify the {{ cssxref("content") }} property with the text content as its value.

Reviewer: I didn't know what was meant by marker until I went and looked at the `::marker` docs. I understand that might be ignorant.

How about:

```suggestion
CSS can insert text content before or after an element, or change the content of a list item marker (such as a bullet symbol or number) before a {{HTMLElement('li')}} or other element with {{ cssxref("display", "display: list-item;") }}. To specify this, make a rule and add {{ cssxref("::before") }}, {{ cssxref("::after") }}, or {{cssxref("::marker")}} to the selector. In the declaration, specify the {{ cssxref("content") }} property with the text content as its value.
```
